R/formula.R

```diff
@@ -91,7 +94,7 @@ flatten_formula <- function(formula) {
 #' @param terms A vector of linear terms as strings.
 #' @return a vector of unique linear individual terms.
 flatten_individual_terms <- function(terms) {
-  if (is.null(terms))
+  if (length(terms) == 0)
```

Comment:
Interestingly, `NULL` does not work here because for some reason `terms` might be `character(0)`. I'll investigate this.